core/bin/zksync_api/src/fee_ticker/validator/mod.rs

```diff
@@ -190,40 +190,40 @@ mod tests {
 
     #[tokio::test]
     async fn check_tokens() {
-        let dai_token_address =
-            Address::from_str("6b175474e89094c44da98b954eedeac495271d0f").unwrap();
-        let dai_token = Token::new(TokenId(1), dai_token_address, "DAI", 18, TokenKind::ERC20);
-        let phnx_token_address =
-            Address::from_str("38A2fDc11f526Ddd5a607C1F251C065f40fBF2f7").unwrap();
-        let phnx_token = Token::new(TokenId(2), phnx_token_address, "PHNX", 18, TokenKind::ERC20);
+        let rif_token_address =
+            Address::from_str("0x2acc95758f8b5f583470ba265eb685a8f45fc9d5").unwrap();
+        let rif_token = Token::new(TokenId(1), rif_token_address, "RIF", 18, TokenKind::ERC20);
+        let rdoc_token_address =
+            Address::from_str("0x2d919f19D4892381d58EdEbEcA66D5642ceF1A1F").unwrap();
```

Comment:
Where does this address come from?

Reply:
is the address in `etc>tokens>mainnet.json`

Reply:
update